pages/users/settings.tsx

```diff
@@ -0,0 +1,86 @@
+import { FunctionComponent, useState } from 'react';
+import Layout from 'components/Layout';
+import styles from 'styles/users/SettingsEdit.module.css';
+import { Button, Link } from '@material-ui/core';
+import CreateIcon from '@material-ui/icons/Create';
+import SettingsSave from 'components/user/save';
+import SettingsEdit from '../../components/user/edit';
+import SampleUser from './sample';
+
+const UserProfSettings: FunctionComponent<{ initial?: string }> = ({
+  initial = 'edit',
+}) => {
+  const [setting, setSetting] = useState(initial);
```

Comment:
initial doesn't need to be a prop
you can initialize state by putting in the `useState` like `useState('edit')`

Reply:
it might be good to actually use integers instead of string